src/index.js

```diff
@@ -1,11 +1,83 @@
 import './style.css';
 import { Nav } from './components.js';
 import Helper from './helper.js';
+import Icon from './assets/images/download_120262.png';
 
 const helper = new Helper();
 
 document.querySelector('header').innerHTML = Nav();
 
 Helper.displayData('a');
 
-document.querySelector('form').addEventListener('submit', helper.searchHandler);
\ No newline at end of file
+document.querySelector('form').addEventListener('submit', helper.searchHandler);
+
+const icons = new Image();
+icons.src = Icon;
+
+setTimeout(() => {
+  const buttons = document.querySelectorAll('.comment-btn');
+  buttons.forEach((element) => {
+    element.addEventListener('click', (e) => {
+      const args = e.target.getAttribute('data-id');
+      Helper.getInfo(args)
+        .then((data) => {
+          const divs = document.querySelector('.section');
+          divs.innerHTML = `
+        <div class="top-section">
+          <span class="close">&times;</span>
+          <div class="img-section">
+            <img src="${data.image.medium}" alt="" class="main-img">
+            <button class=""><p>Download</p><img src="${icons.src}" alt="Refresh_icon"></button>
+          </div>
+          <div class="details">
+            <h3>${data.name}</h3>
+            <p>Year: ${data.ended}</p>
+            <p>${data.genres}</p>
+            <p>Summary: ${data.summary}</p>
+            <p>Language: ${data.language}</p>
+            <p>Rating: ${data.rating.average}</p>
+            <p>Premeired: ${data.premiered}</p>
+          </div>
+        </div>
+        <div class="comment-section">
+          <h2>Comments</h2>
+          <div class="comment-display">
+            <div class="flex commenting"><p>03/11/2021</p><p>Alex: I'd Love to buy it as soon as I can</p></div>
+            <div class="flex commenting"><p>03/12/2021</p><p>Mia: Nice movie</p></div>
+          </div>
+          <form id="form" class="flex direction">
+            <input type="text" id="name" placeholder="Your Name">
+            <textarea id="comment" maxlength="500" placeholder="Your Insight"></textarea>
+            <button type="submit" id="adding">Submit</button>
+          </form>
+        </div>`;
```

Comment:
The code will look modularized if this can be put in the component file, as it is a component

Reply:
You have a flow that you are using. As you've set the groundwork it will take me more time and energy to decouple my code and modularize it the way you want. If I had more time I would love to follow you're system but it's harder for me because I not only have to write code but I also have to figure out how your code works before I can add my own.

Reply:
I've fixed the popup problem